### ISSUE TITLE
CUDA compilation lib path error

### ISSUE THREAD

**Issue by dextorious** [JuliaGPU/GPUArrays.jl#25]
Running the latest tagged versions of CUDAdrv, CUDArt and GPUArrays, `Pkg.test("GPUArrays")` yields (among other things):

```
WARNING: Couldn't find cuda compilation lib in default location.
        This will disable the ability to compile a CUDA kernel from a string
        To fix, install CUDAdrv in default location.
```

The warning (and subsequent test failure) is caused by [https://github.com/JuliaGPU/GPUArrays.jl/blob/d3d54c7a58154cd9727a2af96f01fd0c909a118e/src/backends/cudanative/cudanative.jl#L111](this line) in the backend definition, which refers to a file that was moved from CUDAdrv to CUDArt by [https://github.com/JuliaGPU/CUDAdrv.jl/commit/988cdaceef4126254a10c539782d8009c4c4f58e](this commit). 

Should be a simple three character fix from
`compile_lib = Pkg.dir("CUDAdrv", "examples", "compilation", "library.jl")`
to
`compile_lib = Pkg.dir("CUDArt", "examples", "compilation", "library.jl")`
so didn't seem worthy of a PR.

**Comment by SimonDanisch** [comment]
Yes, thank you! I actually fixed this in the branch I'm working on right now ....
